tests/resources/VCH-Util.robot

```diff
@@ -128,6 +128,54 @@ Get Docker Params
     Run Keyword If  ${port} == 2376  Set Environment Variable  VCH-PARAMS  -H ${dockerHost} --tls
     Run Keyword If  ${port} == 2375  Set Environment Variable  VCH-PARAMS  -H ${dockerHost}
 
+Get Compose Params  ${output}  ${certs}  ${dockerHost}
+
+Get Compose Params
```

Comment:
Please just roll this into Get Docker Params so that appropriate env vars are set for compose.

Also note that you cannot combine command line arguments with env vars - and it's not possible to specify `COMPOSE_TLS_VERSION` via the command line - this means we're entirely dependent on having the environment variables set correctly.